### ISSUE TITLE
ion-footer does not follow the move when the page is transitioning

### ISSUE THREAD

**Issue by lh4111** [driftyco/ionic#9121]
#### Short description of the problem:
The `ion-footer` does not follow the move when the page is transitioning, but its children element are moved

![](http://o93ntudss.bkt.clouddn.com/ion-footer.gif)

[The URL of the original image](http://o93ntudss.bkt.clouddn.com/ion-footer.gif) 

**Which Ionic Version?** 
2.0.0-rc.2

```
Cordova CLI: 6.3.1
Gulp version:  CLI version 3.9.1
Gulp local:
Ionic Framework Version: 2.0.0-rc.2
Ionic CLI Version: 2.1.4
Ionic App Lib Version: 2.1.2
Ionic App Scripts Version: 0.0.39
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Mac OS X Sierra
Node Version: v6.2.2
Xcode version: Xcode 8.1 Build version 8B62
```

**Comment by joaoveiga** [comment]
Try to wrap your code inside the `ion-footer` in a `ion-toolbar`, just like this:
```
<ion-header>
    <ion-navbar color="brand">
        <ion-title>Footer Transition</ion-title>
    </ion-navbar>
</ion-header>

<ion-content>
    Content
</ion-content>

<ion-footer>
    <ion-toolbar>
        This is visible during nav.push(Page) transition
        and does not disappear until transition has completed
    </ion-toolbar>
</ion-footer>
```

**Comment by lh4111** [comment]
@joaoveiga  Thank you, but your solution doesn't work

**Comment by lh4111** [comment]
My solution is modify 
[https://github.com/driftyco/ionic/blob/master/src/transitions/transition-ios.ts#L39](url)
[https://github.com/driftyco/ionic/blob/master/src/transitions/transition-ios.ts#L118](url)

remove  ` > * `  after `ion-footer`seletor

**Comment by jgw96** [comment]
Hello! Thanks for using Ionic! I am going to close this issue as a duplicate of https://github.com/driftyco/ionic/issues/8804 but link your PR to that issue. Thanks for using Ionic!

**Comment by aoe2rtw2** [comment]
Man I struggled for so long with this issue (which appears to have returned or is still around in version 3.9.2).  Tried everything like wrapping the footer content in a toolbar, removing padding, etc.  Eventually I ended up trying w/ the footer content in the ion-content area itself w/ bottom: 0 set (which fixes the transitions), but that led to glitches when I autoscrolled to display new chat messages.  Removing the > * after the ion-footer selectors in the transition code is the only thing I found anywhere that works on all fronts.  Thanks so much @lh4111 !
